bigquery/storage/managedwriter/options.go

```diff
@@ -70,7 +70,7 @@ func WithMaxInflightBytes(n int) WriterOption {
 // This is generally for diagnostic purposes only.
 func WithTraceID(traceID string) WriterOption {
 	return func(ms *ManagedStream) {
-		ms.streamSettings.TraceID = traceID
+		ms.streamSettings.TraceID = buildTraceID(traceID)
```

Comment:
Do you want to allow users to add in their own trace id?

Reply:
This changes from just setting the traceID in total to building a prefix for the library, and the user-specific trace ID.  Same as jsonwriter etc.

Reply:
take a look at options_test.go for the expected output